tokio-threadpool/src/config.rs

```diff
@@ -14,6 +14,7 @@ pub(crate) struct Config {
     pub around_worker: Option<Callback>,
     pub after_start: Option<Arc<dyn Fn() + Send + Sync>>,
     pub before_stop: Option<Arc<dyn Fn() + Send + Sync>>,
+    #[allow(clippy::type_complexity)]
```

Comment:
Can this be fixed or is it a bug w/ clippy? If the second, can we link the issue.

Reply:
This can be fixed by using type alias (https://github.com/tokio-rs/tokio/pull/1335/commits/56a7c33bfe00c95820e224e804d00e9d97ca75c3).